### PR TITLE
OS-8170 size SmartOS USB for smaller cards

### PR DESCRIPTION
**Pull request by ahayes**
Some memory cards such as the Kingston 2GB SDCard that ships in Dell servers have fewer than 2GB of space for writing the SmartOS boot image. I have lowered the image size by 60MB to accommodate cards that fall short.

I have made minimal changes so as to avoid touching the "proforma" workflow or options and have tested building and running from this image on aforementioned cards.

Discussion here: https://smartos.topicbox.com/groups/smartos-discuss/Tecbe189dfeb6ebfc/the-partition-size-is-fixed-1-86g-in-usb-stick-can-it-be-enlarged

### CODE REVIEW

File: tools/build_boot_image [hunk 1]
```diff
@@ -45,7 +45,7 @@ bi_make_iso=0
 bi_proforma_only=0
 bi_nocleanup=0
 bi_imgsz_gb="2"
-bi_imgsz=$(( $bi_imgsz_gb * 1000000000 ))
+bi_imgsz="1940000256"
```
Comment (jlevon) [inline tools/build_boot_image:48]: Please add a comment as to why we have such a weird value.

File: tools/build_boot_image [hunk 2]
```diff
@@ -45,7 +45,7 @@ bi_make_iso=0
 bi_proforma_only=0
 bi_nocleanup=0
 bi_imgsz_gb="2"
```
Comment (jlevon) [inline tools/build_boot_image:47]: Now this no longer reflects reality, please can you:

 - replace "bi_imgsz_gb" with something like "bi_proforma_prefix=""
 - fix up the -p handling to set that to ${OPTARG}gb and bi_imgsz to $OPTARG * 1000000000

File: tools/build_boot_image (hunk 1 above)
Comment (ahayes) [reply tools/build_boot_image:51]: I committed a comment just above the initialization of variables section to explain the value. It seemed to be in keeping with how other documentation was organized. Let me know if you would prefer it on the line directly above or following on the same line instead.

File: tools/build_boot_image (hunk 2 above)
Comment (ahayes) [reply tools/build_boot_image]: Thanks. New commit has replaced variable with new suggested name and fixed up the spots it was used.

**Comment by jlevon** [comment]
@timfoster could you look at IA please?

**Comment by timfoster** [comment]
Yep, looks fine. Thanks for fixing this.

**Comment by jlevon** [comment]
Thanks!

**Pull request merged**